stream/sentinel-intel/src/sentinel_intel_connector/api_handler.py

```diff
@@ -0,0 +1,236 @@
+import requests
+from pycti import OpenCTIConnectorHelper
+
+from .utils import (
+    get_action,
+    get_description,
+    get_expiration_datetime,
+    get_hash_type,
+    get_hash_value,
+    get_ioc_type,
+    get_tags,
+    get_threat_type,
+    get_tlp_level,
+)
+
+
+class SentinelApiHandler:
+    def __init__(self, helper, config):
+        """
+        Init Tanium API handler.
+        :param helper: PyCTI helper instance
+        :param config: Connector config variables
+        """
+        self.helper = helper
+        self.config = config
+
+        self.headers = None
```

Comment:
You can use the requests.Session Object to store efficiently the header and reuse it directly when calling the different endpoints and methods.

Reference : https://docs.python-requests.org/en/latest/user/advanced/